docker-compose.yml

```diff
@@ -1,13 +1,17 @@
 version: "3.9"
+
 services:
   noresm_land_sites_platform:
-    image: "lassetk/noresm_land_sites_platform:latest"
-    mem_limit: 1024m # Max memory used
-    mem_reservation: 256M # Minimum memory used
-    cpus: 1.5 # Number of CPU cores to be used
+    build:
+      context: ./
+    image: "lassetk/noresm_land_sites_platform:${VERSION:-latest}"
```

Comment:
Let's wait for the new organization to be set, and change this to the new address.